src/runtime/index.ts

```diff
@@ -1,7 +1,123 @@
-import platform from "./platform";
+import platform, { Platform } from "./platform";
+import { EventEmitter } from "node:events";
+import { Transport } from "./transport";
+import { MessageEvent } from "../proto/server/v1/realtime";
 
-export class Realtime {
+type SubscribeCallback = (string) => void;
+
+export interface RealTimeConfig {
+  /*
+   * An id used to identify this client when using the presence feature.
+   * If the client id is not set, an error will be thrown when using any presence features
+   */
+  clientId?: string;
+  platform: Platform;
```

Comment:
What is platform?

Reply:
Platform is either browser or nodejs. Later we can also support webworkers and react-native

Reply:
Is this then internal to us? Not settable by the user, is that right?

Reply:
Correct. This helps with the build process so that we build the correct client for nodejs or browser.